CallToPower/BlissLauncher
# Drag drop widget items

You can now swap the order of widgets within the add widgets recycler view. This ordering then gets passed to the home-screen widgets page.

Previously if you wanted to swap the order of widgets you had to remove and re-add in the order you wanted which was a hassle. 

**Scenarios Manually Tested:**

Setup: Start with 3 widgets
- Swap first with last
- Swap last with first
- Swap middle with first
- Swap middle with last
- Swap first with last, then swap last with middle
- Add an additional widget, swap it with the first.
- Add an additional widget, swap it with the first. Add another widget, swap it with the first. 
- Add an additional widget, swap it with the second. Remove the first.
- Add two additional widgets. Swap to second and fourth. Remove all of the original widgets.

**Note** There was a slight issue with the Digital Clock widget. 
It would fail the "_Add an additional widget, swap with first_" test scenario. The widget would not appear in the home-screen, however if you just added the widget it would add as normal and could be swapped after.
I had no issues with any of the other widget types, so it seems to be specific to the Digital Clock and is not a breaking problem.
